sections/Social/InstagramPosts.tsx

```diff
@@ -50,17 +40,13 @@ export async function loader(
   };
 
   return {
-    data: data.slice(0, layout?.numberOfPosts ?? 12),
+    data: data.slice(0, numberOfPosts ?? 5),
```

Comment:
Se a API da erro acredito que data vem como undefined.
```suggestion
    data: data?.slice(0, numberOfPosts ?? 5) ?? [],
```